wordle-web/src/main.ts

```diff
@@ -11,6 +11,17 @@ import 'vuetify/styles'
 import { createVuetify } from 'vuetify'
 import * as components from 'vuetify/components'
 import * as directives from 'vuetify/directives'
+import Axios from 'axios'
+import { Player } from './scripts/userService'
+
+//Check if the app is running on localhost
+if (window.location.hostname === 'localhost' || window.location.hostname === '127.0.0.1') {
+  Axios.defaults.baseURL = 'https://localhost:7053/'
+} else {
+  Axios.defaults.baseURL = 'cscd-379-spring-2023dillon.database.windows.net/'
```

Comment:
The URL here should be the URL to the web service we created for the API, not the address to the database. This might fix the DB issues you're having unless there are problems elsewhere.